xml/abstract_commands.xml

```diff
@@ -165,7 +165,16 @@
             1: Copy data from {\tt arg0} portion of {\tt data} into the
                memory location specified in {\tt arg1}.
         </field>
-        <field name="0" bits="15:0" />
+        <field name="uncached" bits="15">
+            0: Memory accesses go through the hart's local caches just like any
+            normal hart memory access.
+
+            1: Memory accesses bypass the hart's local caches.
```

Comment:
What does this mean? Again, RISC-V doesn’t talk about caches... so not sure exactly what this means for this spec without defining it.

Reply:
I could add:
The definition of "local caches" is implementation-dependent.